### PR TITLE
CHANGELOG.md: Update

### PR DESCRIPTION
**Pull request by t-b**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,13 +16,20 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Fix memory leak in Attribute::get_att_device_class() ([#677][pr-677])
 - Fix crash during alarm state evaluation if attribute value is not set ([#330][i-330], [#681][pr-681])
 - Fix issue where unsubscribing in push_events led to API_EventTimeout ([#686][i-686], [#699][pr-699])
+- Remove event blindness after device restart ([#496][i-496], [#702][pr-702])
+- Include all relevant classes in the C++ doxygen documentation ([#780][pr-780])
+- Fix connection with tango servers with version 9.3.0 in complex network setups ([#716][pr-716])
+- Remove extra comma in struct declaration which trips up old compilers ([#719][pr-719])
```
Comment (bourtemb) [inline CHANGELOG.md]:
```suggestion
- Remove extra comma in struct declaration which trips up old compilers ([#709][pr-709])
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -335,3 +342,13 @@ where a client application might show out of date/incorrect values.
 [pr-689]: https://github.com/tango-controls/cppTango/pull/689
 [pr-690]: https://github.com/tango-controls/cppTango/pull/690
 [pr-699]: https://github.com/tango-controls/cppTango/pull/699
+[i-496]: https://github.com/tango-controls/cppTango/issues/496
+[i-740]: https://github.com/tango-controls/cppTango/issues/740
+[pr-702]: https://github.com/tango-controls/cppTango/pull/702
+[pr-709]: https://github.com/tango-controls/cppTango/pull/709
+[pr-716]: https://github.com/tango-controls/cppTango/pull/716
+[pr-719]: https://github.com/tango-controls/cppTango/pull/719
```
Comment (bourtemb) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -16,13 +16,20 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Fix memory leak in Attribute::get_att_device_class() ([#677][pr-677])
 - Fix crash during alarm state evaluation if attribute value is not set ([#330][i-330], [#681][pr-681])
 - Fix issue where unsubscribing in push_events led to API_EventTimeout ([#686][i-686], [#699][pr-699])
+- Remove event blindness after device restart ([#496][i-496], [#702][pr-702])
```
Comment (bourtemb) [inline CHANGELOG.md:27]:
```suggestion
- Fix Debian7 Travis tests after move of Debian wheezy docker image to debian/eol ([#596][pr-596])
- Remove event blindness after device restart ([#496][i-496], [#702][pr-702])
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -335,3 +342,13 @@ where a client application might show out of date/incorrect values.
 [pr-689]: https://github.com/tango-controls/cppTango/pull/689
 [pr-690]: https://github.com/tango-controls/cppTango/pull/690
 [pr-699]: https://github.com/tango-controls/cppTango/pull/699
+[i-496]: https://github.com/tango-controls/cppTango/issues/496
```
Comment (bourtemb) [inline CHANGELOG.md:353]:
```suggestion
[i-496]: https://github.com/tango-controls/cppTango/issues/496
[pr-596]: https://github.com/tango-controls/cppTango/pull/596
```

**Review by bourtemb — COMMENTED** [review]
#751 seems to be missing too and could probably be added to 
`- Various spelling errors` 
line
I suggested some changes for #596 which seem to be missing and also to fix an error with #709 which was wrongly referring to PR 719.

#780 is currently not in current 9.3.4 tag and is also including by mistake commit d1f422408a22c2ab817ee1d384935ca5268a8c42 which is fixing some timestamp calculation for archive events.

Is there any plan to force-tag again 9.3.4 or do we keep it like that and simply update the Changelog and Doxygen files in 9.3-backports only and regenerate the Doxygen documentation from 9.3-backports branch?

**Comment by t-b** [comment]
@bourtemb Thanks for the review. I've included your changes. And also added an "unreleased" tag with these changes.

> Is there any plan to force-tag again 9.3.4 or do we keep it like that and simply update the Changelog and Doxygen files in 9.3-backports only and regenerate the Doxygen documentation from 9.3-backports branch?

We should not force tag. I can rebuild the doxygen documentation but is that necessary for changelog changes?

**Comment by t-b** [comment]
Merging now for the talk tomorrow.

**Pull request merged**